### ISSUE TITLE
Could you please provide some examples?

### ISSUE THREAD

**Issue by suixin777** [skriegman/evosoro#4]
Could you please provide the example that softrobot escaping from a cage and crossing the barrier as you put on the homepage? Thank you very much.

**Comment by skriegman** [comment]
Yes, I have experimented with generating environments using obstacles (such as the cage, hurdles, etc.). I will try to add this into the repo soon, but I will be busy with research until around February (conference paper season).

In the meantime: 
1. check out the code linked below and search for "cage",
2. compare this to evosoro/tools/read_write_voxelyze.py

https://github.com/ncheney/morphological-innovation-protection/blob/master/cppn/mainSoftbotTwoNetworks.py

**Comment by suixin777** [comment]
Thank you very much.
